### ISSUE TITLE
[Broken] Facebook - can't download any video

### ISSUE THREAD

**Issue by hbtalha** [yt-dlp/yt-dlp#688]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running yt-dlp version **2021.08.10**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] Command-line config: ['https://fb.watch/7lhKTEuki9/', '-v']
[debug] Encodings: locale cp65001, fs utf-8, out utf-8, pref cp65001
[debug] yt-dlp version 2021.08.10 (exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: ffmpeg 4.4, ffprobe 4.4, rtmpdump 2.4
[debug] Optional libraries: mutagen, pycryptodome, sqlite, websockets
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://fb.watch/7lhKTEuki9/
[generic] 7lhKTEuki9: Requesting header
[redirect] Following redirect to https://www.facebook.com/watch/?v=162512749287816
[debug] [facebook] Extracting URL: https://www.facebook.com/watch/?v=162512749287816
[facebook] 162512749287816: Downloading webpage
[facebook] 162512749287816: Downloading webpage
ERROR: Cannot parse data; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1199, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1224, in __extract_info
  File "yt_dlp\extractor\common.py", line 575, in extract
  File "yt_dlp\extractor\facebook.py", line 672, in _real_extract
  File "yt_dlp\extractor\facebook.py", line 592, in _extract_from_url
yt_dlp.utils.ExtractorError: Cannot parse data; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

```

## Description

WRITE DESCRIPTION HERE

Something has changed with Facebook maybe, I haven't been able to download any video today, not even list formats. I tried Instagram and that seems to working fine.

**Comment by pukkandan** [comment]
works for me
```
$ yt-dlp -vF https://fb.watch/7lhKTEuki9
[debug] Command-line config: ['-vF', 'https://fb.watch/7lhKTEuki9']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] yt-dlp version 2021.08.10 (source)
[debug] Plugin Extractors: ['SamplePlugin']
[debug] Git HEAD: d20a7e3c2e
[debug] Python version 3.9.6 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: ffmpeg n4.4-79-gde1132a891-20210809, ffprobe 2021-04-11-git-309e3cc15c-full_build-www.gyan.dev, phantomjs 2.1.1
[debug] Optional libraries: mutagen, pycryptodome, sqlite, websockets
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://fb.watch/7lhKTEuki9
[generic] 7lhKTEuki9: Requesting header
[redirect] Following redirect to https://www.facebook.com/WHOAFRO/videos/162512749287816/
[debug] [facebook] Extracting URL: https://www.facebook.com/WHOAFRO/videos/162512749287816/
[facebook] 162512749287816: Downloading webpage
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 162512749287816:
ID                       EXT  RESOLUTION |   TBR PROTO | VCODEC    VBR ACODEC     ABR  ASR    MORE INFO
------------------------ ---- ---------- - ----- ----- - ------- ----- --------- ---- ------- ---------------------------
267580828134629a         m4a  audio only |   65k https |               mp4a.40.5  65k 48000Hz [eng], DASH audio, m4a_dash
973347706569134v         webm 576x640    |   97k https | vp9       97k                        DASH video, webm_dash
984609198996168v         webm 768x854    |  212k https | vp9      212k                        DASH video, webm_dash
354545502911646v         webm 1080x1200  |  403k https | vp9      403k                        DASH video, webm_dash
dash_sd_src              mp4  unknown    |       https | unknown       unknown
dash_sd_src_no_ratelimit mp4  unknown    |       https | unknown       unknown
dash_hd_src              mp4  unknown    |       https | unknown       unknown
dash_hd_src_no_ratelimit mp4  unknown    |       https | unknown       unknown
```

**Comment by hbtalha** [comment]
Then I don't know what is going on, I still can't download any video from Facebook. 
I notice that yours has a different Python version, I don't if that means something. Or the OS used, I don't know. I am on Windows btw.

**Comment by Fusseldieb** [comment]
> Then I don't know what is going on, I still can't download any video from Facebook.
> I notice that yours has a different Python version, I don't if that means something.

Try to run your command with '--verbose' to see where exactly its failing and how. Might help to find out what's wrong.

**Comment by pukkandan** [comment]
@hbtalha Please run `yt-dlp -v --write-pages https://fb.watch/7lhKTEuki9`, post the console output and attach the `.dump` files that are generated

**Comment by Sipherdrakon** [comment]
I had this happen a lot when trying to download multiple files from Facebook without logging in or cookies. I think it happened after about 15 downloads and then it would throw this error. So might be some sort of soft-ban. Usually reset after 24hrs and then I could get another 15 out before it happened again.

I know that doesn't help but it might provide a starting point to troubleshoot. 

I just downloaded a bunch of stuff from Facebook using ```--cookies-from-browser chrome``` and that used to get my password revoked after a few days. So I'll report back in a few days and let you know if I had to reset my password. If not then you might try passing the cookies in your command and see if that fixes it.
